Makefile

```diff
@@ -80,6 +80,9 @@ logs.%: ## Shows the logs of the given docker service. Example: make logs.datast
 restart.%: ## Restarts the given docker service. Example: make restart.datastore
 	docker compose restart $*
 
+run_tests.lints: ## Runs the linter tests
+	docker compose run --no-deps --entrypoint "python -m scripts.linters.pre_commit_linter $(PYTHON_ARGS)" dev-server
+
```

Comment:
This is fine for now, but I highly recommend running js/ts-related (Eslint) checks on front-end services. 

Also, in the future: Create x-base-backend services which have the core image and no entrypoint and create the child containers with entry-point using it. (Ditto for frontend containers)

Can you please add these to tracking issues?

Reply:
thanks! added this in the https://github.com/oppia/oppia/issues/18638